<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_info</class>
 <widget class="QDialog" name="change_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">color: white;
background-color:rgb(15, 107, 255)</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>48</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">color: white;
background-color:rgb(15, 107, 255)</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>290</width>
     <height>403</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Имя:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Фамилия:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rsurname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Дата рождения:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rbirthday"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Логин:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rlogin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Пароль:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rpassword"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Номер телефона:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rnumber"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Адрес почты:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rmail"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Имя на латинице:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lat_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Фамилия на латинице:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lat_surname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Номер паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pasport_num"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Серия паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pasport_seria"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Срок дествия паспорта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="paspoprt_term"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="user_icon">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mainpicture">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>631</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mainpicure">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>621</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>mainpicure</zorder>
  <zorder>mainpicture</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>user_icon</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>change_info</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
